source/application/REIXS/REIXSAppController.cpp

```diff
@@ -228,6 +227,7 @@ void REIXSAppController::shutdown() {
 
 	// Make sure we release/clean-up the beamline interface
 	AMBeamline::releaseBl();
+	AMStorageRing::releaseStorageRing();
```

Comment:
As per your discussion with @iainworkman earlier, I think you could just put that inside of AMAppController::shutdown().